WTWSMS/decisions/minor_decisions.txt

```diff
@@ -4,17 +4,17 @@ decisions = {
 		only_playable = yes
 		potential = {
 			is_playable = yes
-			OR = {
-				is_feudal = yes
-				is_tribal = yes
-				is_nomadic = yes
-			}
+
 			prisoner = no
-			OR = {
-				religion_group = christian
-				religion_group = zoroastrian_group
-				religion_group = jewish_group
+
+			NOT = { 
+				religion_group = indian_group # They have their own Duwali Feasts
+				religion = zun_pagan # They have their own Sun Feasts
+				religion = zun_pagan_reformed # They have their own Sun Feasts
```

Comment:
The Feast Chain is generic and has nothing special. Everyone who doesn't have their own special Feast can use then ormal feast, because _**ALL**_ rulers held feasts in the Dark Ages and Medieval Period.